### PR TITLE
Add support for he-IL locale

### PR DESCRIPTION
**Pull request by eli-zehavi**

### CODE REVIEW

File: languages/he-IL.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            thousands: ',',
+            decimal: '.'
+        },
+        abbreviations: {
+            thousand: 'אלף',
+            million: 'מליון',
+            billion: 'בליון',
+            trillion: 'טריליון'
+        },
+        ordinal: function () {
```
Comment (BenjaminVanRyseghem) [inline languages/he-IL.js]: You can now remove this function is not needed :smile:

File: tests/languages/he-IL.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+var numbro = require('../../numbro'),
+    language = require('../../languages/he-IL');
+
+numbro.language(language.langLocaleCode, language);
+
+exports['language:he-IL'] = {
+    setUp: function (callback) {
+        numbro.language('he-IL');
```
Comment (BenjaminVanRyseghem) [inline tests/languages/he-IL.js]: replace `he-IL` by `language.langLocaleCode` :smile:

**Comment by BenjaminVanRyseghem** [comment]
Could you please address the minor issues I pointed out?

**Comment by eli-zehavi** [comment]
Hi there, sorry for late answer, I didn't fully understand what are the changes that need to be done.
Please clarify,
Thanks.

File: languages/he-IL.js (hunk 1 above)
Comment (BenjaminVanRyseghem) [inline languages/he-IL.js]: The function `ordinal` should be removed (as the behavior proposed here is also the default implementation)

File: tests/languages/he-IL.js (hunk 2 above)
Comment (BenjaminVanRyseghem) [inline tests/languages/he-IL.js]: replace the line `numbro.language('he-IL');`
by `numbro.language(language.langLocaleCode);`

**Comment by BenjaminVanRyseghem** [comment]
I did another round of commenting, I hope it's clearer to you now :smile:

**Pull request merged**